recurring_cancel.html

```diff
@@ -23,25 +23,25 @@ <h2 class="page-title">{{ page.title }}</h2>
 {% endautoescape %}
 {% for profile in active %}
 
+<div class="section note">
 
 <form class="action_form" name="act" method="POST" action="/act/" accept-charset="utf-8">
 <input type="hidden" name="utf8" value="&#x2714;">
 
 <ul class="compact" id="ak-errors"></ul>
-<div class="section note">
 {% include "./recurring_info.html" %}
+
 <div style="clear:both;"></div>
-<div><button type="submit" class="submit">Cancel Donation</button></div>
-</div>
+
 {% comment %} JavaScript needs it. {% endcomment %}
 <input type="hidden" name="akid" value="{{ logged_in_user.token }}">
 <input type="hidden" name="page" value="{{ page.name }}">
 <input type="hidden" name="profile_id" value="{{ profile.id }}">
 
-<!--<p><input name="submit" type="submit" class="submit" value="Cancel Donation"></p>-->
-
 </form>
 
+<div><button type="submit" class="submit js-modal button bg-blue" data-modal-source="#confirmationPopup">Cancel Donation</button></div>

```

Comment:
Wouldn't this have potential to submit the form since we're not using `preventDefault()`?

Reply:
no, since it's outside the form